emukit/examples/preferential_batch_bayesian_optimization/pbbo/inferences/mcmc_batch_comparison.py

```diff
@@ -0,0 +1,132 @@
+import numpy as np
+import scipy.linalg as la
+import os
+
+import matplotlib.pyplot as plt
+
+
+import itertools
+
+import GPy
+
+from copy import deepcopy
+
+from importlib import reload
+
+from GPy.inference.latent_function_inference.posterior import Posterior
+from GPy.util.linalg import tdot, dpotrs, pdinv, jitchol, dpotri
+import stan_utility
+import os
+import sys
+
+from typing import Tuple, List, Callable, Dict
+# set environmental variable STAN_NUM_THREADS
+# Use 4 cores per chain
+os.environ['STAN_NUM_THREADS'] = "1" # To make our lives easier on clusters
```

Comment:
this should probably be moved to installation/running instructions

Reply:
Yes, good idea!